specs/v2-sketch.yaml

```diff
@@ -0,0 +1,365 @@
+        identities:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              avatar_url:
+                type: string
+                example: "https://avatars.example.com/369ea08d7172bfc72207e2ae1c1b17ef"
+              external_id:
+                anyOf:
+                  - type: string
+                  - type: number
+                example: 1234567
+              id:
+                anyOf:
+                  - type: string
+                  - type: number
+                example: 1234567
+              name:
+                type: string
+                example: "User Name"
+              user?:
+                type: boolean
+              domain:
+                type: string
+                example: "github.com"
+              type:
+                type: string
+                example: "github"
+              authorized?:
+                type: boolean
+              provider_id:
+                type: string
+                format: uuid
+              login:
+                type: string
+                example: "username"
+    Actor:
+      type: object
+      properties:
+        login:
+          type: string
+        name:
+          type: string
+        id:
+          type: string
+          format: uuid
+    Project:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        # add a link to the project, Pat suggested we consider link headers
+    # Elide from Workflow for the time being. If we can get pipelines in weeks we don't need this. Backup plan if people shout is to put this into the Workflow response.
+    TriggerSummary:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: ["webhook", "api", "schedule"]
+        actor:
+          $ref: "#/components/schemas/Actor"
+        vcs:
+          type: object
+          properties:
+            branch:
+              type: object
+              properties:
+                name:
+                  type: string
+                default_branch:
+                  type: boolean
+            tag:
+              type: string
+            commit:
+              type: string
+    Workflow:
+      type: object
+      required:
+        - id
+        - name
+        - created_at
+        - project
+        - errors
+        - status
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        created_at:
+          type: string
+          format: date-time
+        stopped_at:
+          type: string
+          format: date-time
+        # No pipeline entity here. Avoid cycles in the graph
+        pipeline_id:
+          type: string
+          format: uuid
+        pipeline_number:
+          type: number
+          format: integer
+        project:
+          $ref: "#/components/schemas/Project"
```

Comment:
Why inline the project rather than including an ID?

Reply:
I don't envision this being the full project object that would be fetched from a project-specific API. I am pretty sure that it will include some of that data, but haven't rounded that out yet probably:
* our ID
* link to fetch the whole project from our API
* the external URL (e.g. `https://github.com/CircleCI-Public/api-preview-docs`)

My supposition is that should make it easier for the folks building their own dashboards, would love to get your thoughts on that.

Reply:
This feels like an ID and links, which kind of puts us back to the "how are links handled" question above.

In general, this feels like we are optimizing for a very specific use of this API that forces us to join mutable data from another resource into it. I'm at least hesitant to start here. It would be very easy to add and very very hard to remove.

Reply:
There's another PR that adds "links" declarations to the response blocks. I'm about to merge it into this branch.